### ISSUE TITLE
Line chart has a gap between first point and Y-axis

### ISSUE THREAD

**Issue by Nayananga** [h2oai/wave#1582]
#### Wave SDK Version, OS

(e.g. 4.2.0, Windows)
0.21.1, macOS 12.5

#### Actual behavior

(A clear and concise description of what happened.)

The line chart has a gap between the first point and Y-axis

<img width="590" alt="Screenshot 2022-08-10 at 22 01 27" src="https://user-images.githubusercontent.com/25351304/183964814-ad6790f8-4ae7-4e81-8005-ace11236fa10.png">

#### Expected behavior

(A clear and concise description of what you expected to happen.)

There should be a way to remove the gap between the first point and Y-Axis

![Screenshot 2022-08-10 at 15 28 10](https://user-images.githubusercontent.com/25351304/183965000-a3d383c8-d0bb-457e-8520-7901a05f3ae6.png)

#### Steps To Reproduce

minimal working code

```
from h2o_wave import main, data, ui, site

page = site['/demo']

rows = [["2022-08-10 08:20:09", 0.0], ["2022-08-10 08:25:09", 0.63], ["2022-08-10 08:50:09", 0.88]]

marks = [
    ui.mark(
        type="line",
        x="=time",
        y="=feature_drift",
        color="#FEC925",
        x_title="x_title",
        y_title="y_title",
        y_min=0,
        label_overlap="hide",
    ),
    ui.mark(
        y=0.2,
        label=str(0.2),
        label_fill_color="#F44336",
        ref_stroke_color="#F44336",
    )
]

page.add('example', ui.form_card(
    box='1 1 4 4', items=[
        ui.visualization(
            plot=ui.plot(marks),
            data=data("time feature_drift", rows=rows, pack=True),
            interactions=["brush"],
        )
    ]
))

page.save()

```

**Comment by Nayananga** [comment]
Also, I see the line label is crossed by the line itself in the **Safari** web browser

![Screenshot 2022-08-10 at 22 36 11](https://user-images.githubusercontent.com/25351304/183973852-29dc027a-cf4c-4e38-9765-838a99bb43b8.png)

**Comment by mturoci** [comment]
The demo is not runnable for me (see screenshot), but my guess is that the reason for this behavior is `y_min=0`.

<img width="696" alt="image" src="https://user-images.githubusercontent.com/64769322/184084483-bb42e141-e2e3-41eb-8476-c250469fc6d5.png">

**Comment by Nayananga** [comment]
I'm directly copy pasting working code here, please check

```
from h2o_wave import main, data, ui, site

page = site['/demo']

rows = [["2022-08-10 08:20:09", 0.0], ["2022-08-10 08:25:09", 0.63], ["2022-08-10 08:50:09", 0.88]]

marks = [
    ui.mark(
        type="line",
        x="=time",
        y="=feature_drift",
        color="#FEC925",
        x_title="x_title",
        y_title="y_title",
        y_min=0,
        label_overlap="hide",
    ),
    ui.mark(
        y=0.2,
        label=str(0.2),
        label_fill_color="#F44336",
        ref_stroke_color="#F44336",
    )
]

page.add('example', ui.form_card(
    box='1 1 4 4', items=[
        ui.visualization(
            plot=ui.plot(marks),
            data=data("time feature_drift", rows=rows, pack=True),
            interactions=["brush"],
        )
    ]
))

page.save()
```

**Comment by Nayananga** [comment]
@mturoci I tried by removing `y_min=0` but still no difference

**Comment by mturoci** [comment]
You need to use proper ISO dates and set `x_scale='time'` instead of plain strings.

**Comment by mturoci** [comment]
Safari bug fixed in #1587.
